### PR TITLE
Functional props vc em

### PR DESCRIPTION
**Pull request by drfudd2001**
New PR

### CODE REVIEW

**Comment by drfudd2001** [comment]
I made the changes to the .gitignore and deleted the DS_Store from the repo

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
+DS_Store
```
Comment (SunkissedQueen) [inline .gitignore:1]: should be `.DS_Store`

**Review by SunkissedQueen — CHANGES_REQUESTED** [review]
Let's work through this because the package.json is outside of the folder now.

File: .gitignore (hunk 1 above)
Comment (drfudd2001) [reply .gitignore:1]: I made that change but I referred to it using the wrong syntax

**Pull request closed without merging**